src/classes/BDI_DataImportService.cls

```diff
@@ -48,10 +50,24 @@ global with sharing class BDI_DataImportService {
     public BDI_DataImportService(Boolean isDryRun, BDI_FieldMapping fieldMapper) {
         contactService = new BDI_ContactService(this);
         this.isDryRun = isDryRun;
-
         this.fieldMapper = fieldMapper;
     }
 
+    /*******************************************************************************************************
+    * @description queries the Data Import Settings custom setting to determine what type of field mapping to
+    * use.
+    */
+    public static BDI_FieldMapping getDefaultFieldMapping(){
```

Comment:
Just some minor style fixes here - 
* space between ")" and "{" 
* space between "if" and "("
* space between "}" and "else"